tools/ESPEasy-tools.md

```diff
@@ -0,0 +1,150 @@
+
+# ESPEasy tools
+
+This is a set of simple but powerful command line tools for scripting updates, and for backup and restore of configuration settings and ESPeasy rule files
+
+Be careful not to use these tools in a multiuser environment as the login password may display with something like `ps f -ef`
+
+
+## Installation and setup
+
+### Prereqs
+
+You will need:
```

Comment:
What's missing here is:
- A Linux based operating system

😉

Reply:
Good catch / will fix... Bash in WSL other container or virtual environment is also ok

Reply:
Added to requirements, some initial security warnings + a markdown lint